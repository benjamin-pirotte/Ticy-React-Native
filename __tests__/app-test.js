import 'react-native';
import React from 'react';
import AppIos from '../build/index.ios.js';
import AppAndroid from '../build/index.android.js';

// Note: test renderer must be required after react-native.
import renderer from 'react-test-renderer';

it('Ios renders correctly', () => {
  const tree = renderer.create(
    <AppIos />
  ).toJSON();
  expect(tree).toMatchSnapshot();
});

it('Android renders correctly', () => {
  const tree = renderer.create(
    <AppAndroid />
  ).toJSON();
  expect(tree).toMatchSnapshot();
});