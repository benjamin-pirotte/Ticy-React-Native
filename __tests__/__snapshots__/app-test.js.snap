exports[`test Android renders correctly 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "#F5FCFF",
      "flex": 1,
      "justifyContent": "center",
    }
  }>
  <Text
    accessible={true}
    allowFontScaling={true}
    ellipsizeMode="tail"
    style={
      Object {
        "fontSize": 20,
        "margin": 10,
        "textAlign": "center",
      }
    }>
    Welcome to React Native!
  </Text>
  <Text
    accessible={true}
    allowFontScaling={true}
    ellipsizeMode="tail"
    style={
      Object {
        "color": "#333333",
        "marginBottom": 5,
        "textAlign": "center",
      }
    }>
    To get started, edit index.android.js
  </Text>
  <Text
    accessible={true}
    allowFontScaling={true}
    ellipsizeMode="tail"
    style={
      Object {
        "color": "#333333",
        "marginBottom": 5,
        "textAlign": "center",
      }
    }>
    Shake or press menu button for dev menu
  </Text>
  <View
    style={
      Object {
        "marginVertical": 15,
      }
    }>
    <View
      accessibilityComponentType="button"
      accessibilityLabel={undefined}
      accessibilityTraits="button"
      accessible={true}
      hitSlop={undefined}
      onLayout={undefined}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Object {
          "opacity": 1,
        }
      }
      testID={undefined}>
      <Text
        accessible={true}
        allowFontScaling={true}
        ellipsizeMode="tail"
        style={
          Array [
            Object {
              "color": "#007aff",
              "fontFamily": "HelveticaNeue-Medium",
              "fontSize": 17,
              "fontWeight": "bold",
              "textAlign": "center",
            },
            null,
            undefined,
            null,
          ]
        }>
        Press here
         (
        0
        )
      </Text>
    </View>
  </View>
</View>
`;

exports[`test Ios renders correctly 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "#F5FCFF",
      "flex": 1,
      "justifyContent": "center",
    }
  }>
  <Text
    accessible={true}
    allowFontScaling={true}
    ellipsizeMode="tail"
    style={
      Object {
        "fontSize": 20,
        "margin": 10,
        "textAlign": "center",
      }
    }>
    Welcome to React Native
  </Text>
  <Text
    accessible={true}
    allowFontScaling={true}
    ellipsizeMode="tail"
    style={
      Object {
        "color": "#333333",
        "marginBottom": 5,
        "textAlign": "center",
      }
    }>
    To get started, edit index.ios.js
  </Text>
  <Text
    accessible={true}
    allowFontScaling={true}
    ellipsizeMode="tail"
    style={
      Object {
        "color": "#333333",
        "marginBottom": 5,
        "textAlign": "center",
      }
    }>
    This is a first test Press Cmd+R to reload, 
    
    
    Cmd+D or shake for dev menu
  </Text>
  <View
    style={
      Object {
        "marginVertical": 15,
      }
    }>
    <View
      accessibilityComponentType="button"
      accessibilityLabel={undefined}
      accessibilityTraits="button"
      accessible={true}
      hitSlop={undefined}
      onLayout={undefined}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Object {
          "opacity": 1,
        }
      }
      testID={undefined}>
      <Text
        accessible={true}
        allowFontScaling={true}
        ellipsizeMode="tail"
        style={
          Array [
            Object {
              "color": "#007aff",
              "fontFamily": "HelveticaNeue-Medium",
              "fontSize": 17,
              "fontWeight": "bold",
              "textAlign": "center",
            },
            null,
            undefined,
            null,
          ]
        }>
        Press here
         (
        0
        )
      </Text>
    </View>
  </View>
</View>
`;
